skimage/feature/_hog.py

```diff
@@ -145,8 +128,8 @@ def hog(image, orientations=9, pixels_per_cell=(8, 8),
             for y in range(n_cellsy):
                 for o in range(orientations):
                     centre = tuple([y * cy + cy // 2, x * cx + cx // 2])
-                    dx = radius * cos(float(o) / orientations * np.pi)
-                    dy = radius * sin(float(o) / orientations * np.pi)
+                    dx = radius * np.cos(float(o) / orientations * np.pi)
+                    dy = radius * np.sin(float(o) / orientations * np.pi)
```

Comment:
Note I've corrected my own recommendations for the visualization loop - see the updated note on GitHub, don't go by what was emailed out!